docs/data/date-pickers/time-range-picker/time-range-picker.md

```diff
@@ -0,0 +1,18 @@
+---
+product: date-pickers
+title: React Time Range Picker component
+components: TimeRangePicker, DesktopTimeRangePicker, MobileTimeRangePicker, StaticTimeRangePicker
+githubLabel: 'component: TimeRangePicker'
+packageName: '@mui/x-date-pickers'
+materialDesign: https://material.io/components/date-pickers
```

Comment:
Without `{{"component": "modules/components/ComponentLinkHeader.js"}}` this does nothing (also true for the other pages)

Reply:
I never worked with `ComponentLinkHeader`, do you think we should add it or do you think we should remove the useless headers ?